securedrop/static/js/journalist.js

```diff
@@ -52,39 +52,13 @@ $(function () {
     checkboxes.prop('checked', false);
   });
   unread.click(function() {
-      var checkboxes = document.querySelectorAll(".submission > [type='checkbox']");
-      for (var i = 0; i < checkboxes.length; i++) {
-          if (checkboxes[i].className.includes("unread-cb"))
-              checkboxes[i].checked = true;
-          else
-              checkboxes[i].checked = false;
-      }
-    });
-
-  $("#delete-collection").submit(function () {
-    return confirm(get_string("collection-delete-confirm-string"));
-  });
-
-  $("#delete-collections").click(function () {
-    var checked = $(".panel ul#cols li :checkbox").filter(":visible").filter(function(index) {
-        return $(this).prop('checked');
-    });
-    if (checked.length > 0) {
-      return confirm(get_string("collection-multi-delete-confirm-string").supplant({ size: checked.length }));
+    var checkboxes = document.querySelectorAll(".submission > [type='checkbox']");
```

Comment:
Nit: there is no need to change spacing here and that clarifies this part of the code is left untouched by this PR. Not a blocker for merge at all :-)

Reply:
Sure